recipes/levenshtein-search/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "levenshtein-search" %}
+{% set version = "1.4.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Levenshtein_search-{{ version }}.tar.gz
+  sha256: 4034569cb652b00a928f5417bf312c3935a9fd493759ff116d779c44f2f437e2
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - levenshtein_search
```

Ugh, it's [capitalized](https://github.com/mattandahalfew/Levenshtein_search#usage):
```suggestion
    - Levenshtein_search
```